src/sentry/features/__init__.py

```diff
@@ -256,6 +256,7 @@
 default_manager.add("organizations:ds-sliding-window-org", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
 default_manager.add("organizations:pr-comment-bot", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
 default_manager.add("organizations:ds-org-recalibration", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+default_manager.add("organizations:slack-use-new-lookup", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
```

Comment:
You'll want this to be `FeatureHandlerStrategy.REMOTE` so that we can enable the feature using Flagr. https://develop.sentry.dev/feature-flags/#add-your-feature-to-the-featuremanager 

I can help you set up the feature flag in Flagr too :)